modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-builder/src/main/java/com/liferay/dynamic/data/mapping/form/builder/internal/servlet/DDMDataProviderInstancesServlet.java

```diff
@@ -145,9 +146,9 @@ private JSONArray _getDataProviderInstancesJSONArray(
 
 			return dataProviderInstancesJSONArray;
 		}
-		catch (PortalException portalException) {
+		catch (Exception exception) {
 			if (_log.isDebugEnabled()) {
-				_log.debug(portalException);
+				_log.debug(exception);
 			}
```

Comment:
@icgsbr do we still need this `catch` here?

Reply:
I'll take a look right away

Reply:
It works well